src/concrete/ml/sklearn/base.py

```diff
@@ -1766,6 +1767,8 @@ def __init__(self, n_bits: int = 3):
         self._y: numpy.ndarray
         # _q_fit_X_quantizer: The quantizer to use for quantizing the model's training set
         self._q_fit_X_quantizer: Optional[UniformQuantizer] = None
+        # Use the rounding feature
+        self._rounder = cnp.AutoRounder(target_msbs=DEFAULT_ROUNDING_THRESHOLD_BITS)
```

Comment:
question (and I see you more or less started the discussion in the PR's description) : should users still be able to modify the number of bits to use ? If not, fine with me then. But if so, maybe we could add a method like `update_auto_rounder` or something. 

Anyway, if we start to integrate the rounding feature in built-in models, we should indeed decide what we want to do, maybe you can start a discussion on slack about it and we'll talk at the next weekly !

Reply:
No, the user won't be able to update this value.
The idea in testing/impact_of_rouning_on_knn was to find an interesting default rounding value.
We will definitely talk about it in the weekly.

Reply:
got it thanks !